public/category-sitemap.xml

```diff
@@ -1,299 +1,310 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Looks like the `<?xml ...?>` tag has disappeared from all of these docs. Is that deliberate? Otherwise :+1: